content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,271 @@ Resumé tab
 this section may be critical for the company's payroll department. In order to properly process
 payslips and ensure all deductions are accounted for, the employee's personal information should be
 entered.
 
-Here, the employee's :guilabel:`Private Contact`, :guilabel:`Marital Status`, :guilabel:`Emergency
-Contact`, :guilabel:`Education`, :guilabel:`Citizenship`, :guilabel:`Dependant`, and :guilabel:`Work
-Permit` information is entered. Fields are entered either using a drop-down menu, clicking a check
-box, or typing in the information.
-
-- :guilabel:`Private Contact`: Enter the :guilabel:`Address` for the employee. The selection can be
-  made with the drop-down menu. If the information is not available, type in the name for the new
-  address. To edit the new address, click the :guilabel:`External Link` button to open the address
-  form. On the address form, enter the necessary details, then click :guilabel:`Save`. Some other
-  information in the :guilabel:`Private Contact` section may auto-populate, if the address is
-  already listed in the drop-down menu.
-- :guilabel:`Marital Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
-  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
-  menu.
-- :guilabel:`Emergency Contact`: Type in the name and phone number of the employee's emergency
-  contact.
+Here, the employee's :guilabel:`Private Contact`, :guilabel:`Education`, :guilabel:`Work Permit`,
+:guilabel:`Family Status`, :guilabel:`Emergency Contact`, and :guilabel:`Citizenship` information is
+entered. Fields are entered either using a drop-down menu, clicking a check box, or typing in the
+information.
+
+- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
+  selection can be made with the drop-down menu. If the information is not available, type in the
+  name for the new address. To edit the new address, click the :guilabel:`Internal Link` button to
+  open the address form. On the address form, enter or edit the necessary details.
+
+  Some other information in the :guilabel:`Private Contact` section may auto-populate, if the
+  address is already listed in the drop-down menu.
+
+  Enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If the bank is not
+  already configured (the typical situation when creating a new employee) enter the bank account
+  number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account Number` form
+  appears. Fill in the information, then click :guilabel:`Save & Close`.
+
+  Then, select the employee's preferred :guilabel:`Language` from the drop-down menu. Next, enter
+  the :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
+  receiving any type of commuter benefits. Last, enter the employee's license plate information in
+  the :guilabel:`Private Car Plate` field.
```

Comment:
I don't see this field present

Reply:
Private Car Plate first appears in 16.1, so I feel it is universal enough to keep in there.